README.md

```diff
@@ -1,8 +1,44 @@
 # Plex Auto Collections
+##### Version 2.0
```

Comment:
Let's decide on a versioning strategy? I'm partial to semver but I don't have strong feelings. I'd be down for just dates. Something where users can quickly see any breaking changes when they update.

Reply:
to be completely honest i dont really know any versioning strategies but i saw you post that it would be a good idea so i figured id just add in version 2.0 to the git readme and as a printout when you start the program to identify which version you're using. I'm personally aginst using dates over just version numbers but thats mainly because i really can't think of any software that uses dates over versions to determine releases

Reply:
+1 for semver, but no strong feelings here either.

Reply:
semver seems the norm to me with this pull we could start 2.0.0